README.md

```diff
@@ -19,15 +19,19 @@ The project involves developing a POS system user interface and an ECommerce web
 ### Extremely Stretchness Goalmericas
 **- Stripe Integration:** The feature allows the use of credit/debit cards transaction on the POS system and website
 
-### Installation
+## Running the Project
 #### Requirements:
 - Node.js
   
-- PostgreSQL
+- PostgreSQL and psql
```

Comment:
Is PostgreSQL not the same as psql?

Reply:
ah, I was referring to the psql as the CLI, but I think we might not need to add it in since you get it when installing postgres